feat: allow partial setting of window bounds (#15677)
electron/electron: docs/api/browser-window.md

* `bounds` [Rectangle](structures/rectangle.md)
* `animate` Boolean (optional) _macOS_

> Resizes and moves the window to the supplied bounds. Also allows setting any of the `x`, `y`, `width`, or `height` properties, instead of setting them all at once. Properties that are not set will fall back to their current values.

Reviewer: ```suggestion
Resizes and moves the window to the supplied bounds. Any properties that are not supplied will default to their current values.
```

```javascript
const { BrowserWindow } = require('electron')

> let win = new BrowserWindow()

Reviewer 2: `const`
No a big deal though.
